docs/installing/manual_install.md

```diff
@@ -728,6 +762,18 @@ production server**. Replace `www.example.com` with your domain name.
       -subj /CN=www.example.com
     chmod 640 /etc/ssl/certs/www.example.com.cert
 
+If you are running a thin cluster, you will need to edit the top section of
+your <code>nginx-ssl.conf.example</code> file so that it has a `server` line
```

Comment:
Probably want to edit the generated nginx.conf, rather than the example which is checked in to source control